### DISCUSSION TITLE
默认登陆三种协议Code45 使用签名服务器？

### DISCUSSION THREAD

**Discussion by NoNameGMM** [LagrangeDev/Lagrange.Core#340] [Q&A] [answered]
非常抱歉又来问问题（）

BotConfig botConfig = new BotConfig();
botConfig.CustomSignProvider

是否需要使用https://sign.lagrangecore.org/api/sign进行签名
&该..怎么用

**Comment by ghost** [comment, marked as answer]
参考 [LinuxSigner](https://github.com/KonataDev/Lagrange.Core/blob/a720deef1806138688000d43c2e72efd1a246911/Lagrange.Core/Utility/Sign/LinuxSigner.cs#L10)

**Reply by NoNameGMM** [reply]
正常Linux登录时 没有任何关于签名的日志，那是不是就是用[签名服务器](https://sign.lagrangecore.org/api/sign)签名成功了.?
我看了眼[Linux Signer](https://github.com/KonataDev/Lagrange.Core/blob/master/Lagrange.Core/Utility/Sign/LinuxSigner.cs)

**Reply by ghost** [reply]
是的

**Reply by NoNameGMM** [reply]
那为什么还会....45..禁止登陆
用不了吗.?

**Reply by ghost** [reply]
目前 https://sign.lagrangecore.org/api/sign 只有 Linux 的协议
